chain/x/zoracle/handler_test.go

```diff
@@ -277,11 +275,13 @@ func TestRequestInvalidDataSource(t *testing.T) {
 }
 
 func TestRequestWithPrepareGasExceed(t *testing.T) {
-	// Setup test environment
+
 	ctx, keeper := keep.CreateTestInput(t, false)
 
 	ctx = ctx.WithBlockHeight(2)
 	ctx = ctx.WithBlockTime(time.Unix(int64(1581589790), 0))
+	// gas consumed before prepare = 1732482
```

Comment:
What does this mean? Gas consumed before prepare??

I think you should make clear how much gas the code's prepare function is expected to consume. Then test with both with enough and not-enough gas.